problems/variables.js

```diff
@@ -16,6 +22,14 @@
 //  * Calculate how many snacks you would eat total, from your current age until the maximum age.
 //  * Log the result to the screen like this: "You will need `snackNumber` to last you until the age of `maxAge`".
 
+// let currentAge = 42;
+// const maxAge = 100;
+// let snack = 3;
+// let totalSnacks = (maxAge - currentAge) * 3
```

Comment:
Don't hardcode the number of snacks, use the variable.

Reply:
Perfect!! Noted!!